packages/cmr-client/templates/README.hbs

```diff
@@ -0,0 +1,35 @@
+# @cumulus/cmr-client
```

Comment:
It would be helpful if somewhere in this document (and in the cmrjs README), you called out the difference between the two packages and what should go in where so when we're developing we have some clarity/